src/t8_schemes/t8_default/t8_default_vertex/t8_default_vertex.cxx

```diff
@@ -310,11 +310,10 @@ t8_default_scheme_vertex::element_get_reference_coords ([[maybe_unused]] const t
 
 #ifdef T8_ENABLE_DEBUG
 int
-t8_default_scheme_vertex::element_is_valid (const t8_element_t *elem)
+t8_default_scheme_vertex::element_is_valid ([[maybe_unused]] const t8_element_t *elem)
 
 {
-  const t8_dvertex *v = (const t8_dvertex_t *) elem;
-  return 0 <= v->level && v->level <= T8_DVERTEX_MAXLEVEL;
+  return 1;
```

Comment:
```suggestion
  /* A vertex is always valid, since it only saves the level as uint8, which therefore automatically is>= 0 and <= 255 (=MAXLEVEL) */
  return 1;
```